compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -444,8 +444,12 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
   def desugarIdent(i: Ident): Option[tpd.Select] = {
```

Comment:
I could not get rid of the `Option` here since the backend expects this signature